quantum/process_keycode/process_combo.c

```diff
@@ -528,8 +534,9 @@ static bool process_single_combo(combo_t *combo, uint16_t keycode, keyrecord_t *
 }
 
 bool process_combo(uint16_t keycode, keyrecord_t *record) {
-    bool is_combo_key          = false;
-    bool no_combo_keys_pressed = true;
+    uint8_t layer;
+    bool    is_combo_key          = false;
+    bool    no_combo_keys_pressed = true;
```

Comment:
If the change to declare `ref_layer` is done, this block can then be reverted to its original state.
```suggestion
    bool is_combo_key          = false;
    bool no_combo_keys_pressed = true;
```
Also in the `COMBO_ONLY_FROM_LAYER` case the compiler would probably complain about the unused `layer` variable declared here.